saleor/static/dashboard/js/components/google-preview.js

```diff
@@ -0,0 +1,66 @@
+const $seoTitle = $('#seo_title');
+
+const $nameId = $seoTitle.data('bind');
+const $name = $(`#${$nameId}`);
+
+const $seoDescription = $('#seo_description');
+const $descriptionMaterialize = $seoDescription.data('materialize');
+if ($descriptionMaterialize) {
+  var $description = $(`.materialize-textarea[name='${$descriptionMaterialize}']`);
+} else {
+  const $descriptionId = $seoDescription.data('bind');
+  var $description = $(`#${$descriptionId}`);
+}
+
+const $googleTitlePreview = $('#google-preview-title');
+const $googleDescriptionPreview = $('#google-preview-description');
+const $preview = $('.preview');
+const $previewErrors = $('.preview-error');
+
+const watchedEvents = 'input propertychange cut paste copy change';
+
+function truncate(text, seoField) {
+  const $fieldMaxLength = seoField.prop('maxLength');
+  if ($fieldMaxLength === -1) {
+    console.log('Field maxlength is not defined');
```

Comment:
`console.warn()` appears to be more suited to display this message

Reply:
Any console logs or warnings should be considered as debug rather than a solution.